development/metamaskbot-build-announce.js

```diff
@@ -243,6 +246,61 @@ async function start() {
     console.log(`No results for ${summaryPlatform} found; skipping benchmark`);
   }
 
+  try {
+    const prBundleSizeStats = await (
```

Comment:
We don't need to fetch this; it's available locally. Take a look at the `benchmarkPath` variable for an example of how to reference local test artifacts